packages/babel-preset/namespace-plugin.js

```diff
@@ -29,12 +29,12 @@ function getPrefixFromPackage(filename) {
     }
   }
 
-  const { path, pkg } = readPkgUp.sync({ cwd: dirname(filename) }) || {};
+  const pkgUpResult = readPkgUp.sync({ cwd: dirname(filename) });
+  if (!pkgUpResult) return '';
 
-  if (!path) return '';
-  const prefix = `${pkg.name}:`;
-  PREFIXES.set(dirname(path), prefix);
-  return '';
```

Comment:
@jquense I think this was a bug? I don't see why we'd want to return an empty string for the prefix on the first hit.

Reply:
oops